baseband/vdif/header.py

```diff
@@ -74,13 +110,13 @@ def __new__(cls, words, edv=None, verify=True, **kwargs):
 
         # Have to use key "-1" instead of "False" since the dict-lookup treats
         # 0 and False as identical.
-        cls = cls._vdif_edv_header_classes.get(edv if edv is not False else -1,
-                                               VDIFBaseHeader)
+        cls = _VDIFHeaderRegistry._vdif_edv_header_classes.get(
+                               edv if edv is not False else -1, VDIFBaseHeader)
         return super(VDIFHeader, cls).__new__(cls)
 
     def __init__(self, words, edv=None, verify=True, **kwargs):
-        if edv is not None:
-            self.edv = edv
+        #if edv is not None:
```

Comment:
You cannot remove this - right now, for unknown `edv`, you get the default `VDIFBaseHeader`, which does not have `cls._edv` set, but does need to set `self._edv` since otherwise the verification will fail. So, just change `self.edv = edv` to `self._edv = edv`.

And since this didn't cause a test failure, we better add a test that if you do, e.g.,
```
h = VDIFHeader.fromvalues(edv=100)
```
the following will be true:
```
type(h) is VLBIBaseHeader
h['edv'] == 100
```
(It currently is true)